src/middlewared/middlewared/plugins/account.py

```diff
@@ -2027,5 +2028,13 @@ async def __common_validation(self, verrors, data, schema, pk=None):
 
 
 async def setup(middleware):
+    try:
+        await middleware.call('filesystem.set_zfs_attributes', {
+            'path': DEFAULT_HOME_PATH,
+            'zfs_file_attributes': {'immutable': True}
+        })
+    except Exception:
+        middleware.logger.error('Failed to set immmutable property on /var/empty', exc_info=True)
```

Comment:
First typo: immmutable -> immutable

Second, I know it equates to the same thing, but just to future-proof, would this be better?
```
 middleware.logger.error('Failed to set immutable property on %r', DEFAULT_HOME_PATH, exc_info=True)

```